source/transfer_list.rst

```diff
@@ -157,28 +191,18 @@ The *tag_id* ranges are described in :numref:`tab_tag_id_ranges`.
    * - tag ID range
      - Description
 
-   * - 0x0 -- 0xf_ffff
-     - Standard tag id range. Any tag id in this range must first be allocated in this specification before being used. The allocation of the tag id requires the entry layout to be defined as well.
-
+   * - 0x0 -- 0xff_efff
+     - Standardized range. Any tag ID in this range must first be allocated in this specification before being used. The allocation of the tag ID requires the entry layout to be defined as well.
```

Comment:
I agree with putting the standardized range first but I thought it would be good to retain some reserved bits after this, either for the future special usage you mentioned before or to extend the standardized range when needed.

Reply:
Will do.

Reply:
Done in latest update.